Documentation/contributing.rst

```diff
@@ -1103,60 +1164,40 @@ example patch that shows how this can be achieved.
 
                             //This test should run in each PR for now.
 
-Jenkins Job Descriptions
-------------------------
-
-Cilium-PR-Ginkgo-Tests-Validated
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
-
-The configuration for this job is contained within ``ginkgo.Jenkinsfile``.
-
-It first runs unit tests using docker-compose using a YAML located at
-``test/docker-compose.yaml``.
-
-The next steps happens in parallel:
-
-    - Runs the runtime e2e tests.
-    - Runs the Kubernetes e2e tests against the latest default version of Kubernetes specified above.
-
-Cilium-Nightly-Tests-PR
-~~~~~~~~~~~~~~~~~~~~~~~
-
-The configuration for this job is contained within ``Jenkinsfile.nightly``.
-
-This first runs the Nightly tests with the following setup:
-
-    - 4 Kubernetes 1.8 nodes
-    - 4 GB of RAM per node.
-    - 4 vCPUs per node.
-
-Then, it runs tests Kubernetes tests against versions of Kubernetes that are currently not tested against
-as part of each pull-request, but that Cilium still supports.
-
-It also runs a variety of tests against Envoy to ensure that proxy functionality is working correctly.
+Triage Process
+~~~~~~~~~~~~~~
 
-Cilium-PR-Ginkgo-Tests-k8s
-~~~~~~~~~~~~~~~~~~~~~~~~~~
+Test failures reported on https://jenkins.cilium.io/ should be handled in the following way:
 
-Runs the Kubernetes e2e tests against all Kubernetes versions that are not currently not tested as part
-of each pull-request, but which Cilium still supports, as well as the the most-recently-released versions
-of Kubernetes that are not yet declared stable by Kubernetes upstream:
+Git Branch HEAD CI
+^^^^^^^^^^^^^^^^^^
 
-First stage (stable versions which Cilium still supports):
+This includes the following pipelines:
 
-    - 1.7
-    - 1.8
+- https://jenkins.cilium.io/job/cilium-ginkgo/job/cilium/job/master/
+- https://jenkins.cilium.io/job/cilium-ginkgo/job/cilium/job/v1.0/
 
-Second stage (unstable versions)
+1. Investigate the failure
+2. If false negative, search for existing GitHub issue covering the reason for
+   the false negative. If unknown, open new GitHub issue with prefix CI: and label
+   kind/bug/CI
+3. If not a false negative, open GitHub issue and assign label release-blocker
+   if applicable
+4. Adjust description (Click on build -> Edit Build Information)
```

Comment:
I would note that only people who have accounts authorized to log onto Jenkins are able to do this.